tests/src/rules/order.js

```diff
@@ -2258,14 +2302,14 @@ context('TypeScript', function () {
               parser,
               options: [
                 {
-                  groups: ['external', 'index'],
+                  groups: ['external', 'index', 'type'],
```

Comment:
confirming: this was the test you added, thus, the only impact is on the feature from that PR?

Reply:
yes, I reverted the test changes from the previous commit and added new ones for the type group